tests/date/test_comparison.py

```diff
@@ -137,20 +137,20 @@ def test_less_than_or_equal_false():
     assert not d1 <= d3
 
 
-def test_is_birthday():
+def test_is_anniversary():
```

Comment:
You won't have any tests on `is_birthday` any more if you do this

Reply:
I think that if these two methods are equal, I don't need additional tests. But if you think differently, I can duplicate them or write the additional ones. What is your opinion?

Reply:
I duplicated the tests for `is_birthday` to avoid changes provides by mistake.

Reply:
As far as the unit tests are concerned, it doesn't know that they're equal. So I think your options are:

- duplicate the tests (as you've done)
- test attributes are equal (this may not be a good idea, since you may want to change one or the other later)
- parameterize tests to run the same tests on both methods.